docs/app/pages/components/components-sections/date-time-picker/date-range-picker/date-range-picker.component.html

```diff
@@ -172,6 +172,12 @@
     <tr uxd-api-property name="endPickerAriaLabel" type="string" defaultValue="Selecting the end date">
         Defines the aria-live announcement when a date in the end picker is focused.
     </tr>
+    <tr uxd-api-property name="dateLocalization" type="string | Function">
+        Allows localization of the start/end date strings.
+    </tr>
+    <tr uxd-api-property name="timeLocalization" type="string">
```

Comment:
Can't this also be a function. We should maybe also tell them what format the string can be? Perhaps even link to the Angular Date Pipe formats section?